pkg/controller/controller.go

```diff
@@ -216,3 +223,22 @@ func (c *Kleaner) DeletePod(pod *corev1.Pod) {
 	}
 	metrics.GetOrCreateCounter(metricName(podDeletedMetric, pod.Namespace)).Inc()
 }
+
+// In Case If Pod(s) is Stuck in Terminating state just in case to delete with force
+// Not goood way to fid the root cause of the issue why pod is stuck in terminating state
+func (c *Kleaner) DeletePodWithForce(pod *corev1.Pod) {
```

Comment:
as we discussed in the [issue ](https://github.com/lwolf/kube-cleanup-operator/issues/88#issuecomment-1237386994) I'd prefere not to have `force-deletion` in the codebase

Reply:
@lwolf  yes but we can keep by enabling the flag. it will be useful  when there is an issue calico and workload on this node will be blocked due to this issue so to avoid those pod(s) which are stuck in terminating state until the real issue is fixed

Reply:
no, the issue should be solved by node draining, not force-deletion.